.github/workflows/fips-checksums.yml

```diff
@@ -73,3 +73,29 @@ jobs:
         with:
           name: fips_checksum
           path: artifact/
+  verify-checksums:
```

Comment:
If this is busted and we don't care, should we remove it?

Reply:
Actually this seems to have worked?

Reply:
Still failed, but it's something we can look at: removing some of these CI actions.